src/applications/vaos/services/mocks/v2/confirmed.json

```diff
@@ -1354,6 +1354,10 @@
           "url": "https://care2.evn.va.gov/vvc-app/?join=1&media=1&escalate=1&conference=VAC00064b6f@care2.evn.va.gov&pin=4569928835#",
           "atlas": null,
           "vvsKind": "CLINIC_BASED"
+        },
+        "claim": {
+          "id": "987654323",
+          "status": "Claim Paid"
```

Comment:
This is subject to change - we're not sure yet if we'll be appending the claim status as well, depending on final designs. May I ask which designs you're basing this on (so we're all on the same page, I've been waiting for finalized designs before finalizing the API association logic).

Reply:
Okay that's fine if I see an ID I think that will be enough for the proposed UI here: https://www.figma.com/design/RzugGEmu4drhCSHTyQ6hjl/Simple-mileage-only-travel-pay-claim-submission?node-id=2135-3&node-type=canvas&t=jbwHEvpeN63cY1Dw-0